packages/forms/frontend/sirius-components-forms/src/propertysections/LinkPropertySection.tsx

```diff
@@ -11,16 +11,47 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 import Link from '@material-ui/core/Link';
+import { makeStyles, Theme } from '@material-ui/core/styles';
 import { LinkPropertySectionProps } from './LinkPropertySection.types';
+import { PropertySectionLabel } from './PropertySectionLabel';
+import { getTextDecorationLineValue } from './getTextDecorationLineValue';
+interface StyleProps {
+  color: string | null;
+  fontSize: number | null;
+  italic: boolean | null;
+  bold: boolean | null;
+  underline: boolean | null;
+  strikeThrough: boolean | null;
+}
 
+const useStyle = makeStyles<Theme, StyleProps>(() => ({
+  style: {
+    color: ({ color }) => (color ? color : 'inherit'),
+    fontSize: ({ fontSize }) => (fontSize ? fontSize : 'inherit'),
+    fontStyle: ({ italic }) => (italic ? 'italic' : 'inherit'),
+    fontWeight: ({ bold }) => (bold ? 'bold' : 'inherit'),
+    textDecorationLine: ({ underline, strikeThrough }) => getTextDecorationLineValue(underline, strikeThrough),
+  },
+}));
```

Comment:
This changes the appearance of the existing usage of the link widget. I'll fix that.